beacon_node/beacon_chain/tests/block_verification.rs

```diff
@@ -1087,7 +1087,7 @@ async fn block_gossip_verification() {
     assert!(
         matches!(
             unwrap_err(harness.chain.verify_block_for_gossip(Arc::new(block.clone())).await),
-            BlockError::BlockIsAlreadyKnown,
+            BlockError::BlockIsAlreadyKnown(_),
```

Comment:
Might be worth matching for block root since we have it?

Reply:
you can't use `matches!` with a value in the pattern you're matching, so I tried to swap this for `assert_eq!` but it requires deriving `PartialEq` on a bunch of types (which is why I think this assert is written as it is)

Reply:
All good, that was a nitpick - was almost tempted to delete it lol